apps/arweave/src/ar_storage_module.erl

```diff
@@ -124,8 +124,9 @@ packing_label(Packing) ->
 
 %% @doc Return the storage module with the given identifier or not_found.
 %% Search across both attached modules and repacked in-place modules.
-get_by_id(?DEFAULT_MODULE) ->
-	?DEFAULT_MODULE;
+get_by_id(Atom) when is_atom(Atom) ->
```

Comment:
`?DEFAULT_MODULE` is `"default"` rather than `default` - will this still handle that case correctly?

Reply:
Hm, good catch, I think I removed it by mistake